ts/parts/AreaSeries.ts

```diff
@@ -520,6 +530,13 @@ seriesType<Highcharts.AreaSeries>(
             }
 
             topPath = getGraphPath.call(this, graphPoints, true, true);
+            if (series.chart.is3d()) {
+                bottomPoints = H.perspective(
+                    bottomPoints as any, this.chart, true
+                ).map(function (point): Highcharts.AreaPoint {
+                    return { plotX: point.x, plotY: point.y, plotZ: point.z } as any;
+                }) as any;
```

Comment:
Too many casts "as any". What is the reason of not creating an interface of them or use existed i.e Points?

Reply:
I am not sure if you are aware that it is still Draft :) The first PR is made to check if all visual test are passing with new series approach.

Reply:
Mea culpa, just waiting for a final version :) Sorry!

Reply:
However, I think that in general problem of casting to any in Area series is something we should work on. I thought about creating simple interface of point positions as `{x?, y?, z?, plotX?, plotY?, plotZ?}` that will solve needs of both standard and 3d graph in this case.

Reply:
All is moved into new file and casting is significantly reduced.